meta-lxatac-software/recipes-core/lxatac-profile/lxatac-profile/01-labgrid.sh

```diff
@@ -2,4 +2,4 @@
 
 # export labgrid environment to be used with labgrid-client on the lxatac
 source /etc/labgrid/environment
-export LG_CROSSBAR="ws://${LABGRID_COORDINATOR_IP:?}:${LABGRID_COORDINATOR_PORT:?}/ws"
+export LG_COORDINATOR="${LABGRID_COORDINATOR_IP:?}:${LABGRID_COORDINATOR_PORT:?}"
```

Comment:
Please add a note to the commit message that we leave the two variables separate, so we can remember in a few years why it is this way.

Reply:
Sounds good. Done.